WhateverGreen/kern_igfx.cpp

```diff
@@ -593,6 +605,10 @@ bool IGFX::processKext(KernelPatcher &patcher, size_t index, mach_vm_address_t a
 			}
 		}
 
+		// We could iterate through each submodule and redirect the request if and only if the submodule is enabled
+		if (modDVMTCalcFix.enabled)
+			modDVMTCalcFix.processFramebufferKext(patcher, index, address, size);
```

Comment:
I believe a similar call to graphics kext is missing.

Reply:
Yes, but not here. Fixed.